### PR TITLE
zh-CN: updated the translation of `tab-size`

### PR DESCRIPTION
**Pull request by daiwanxing**

### CODE REVIEW

File: files/zh-cn/web/css/tab-size/index.md [hunk 1]
```diff
@@ -3,70 +3,72 @@ title: tab-size
 slug: Web/CSS/tab-size
 ---
 
-{{CSSRef}}{{SeeCompatTable}}
+{{CSSRef}}
 
-CSS 属性 **`tab-size`** 用于自定义制表符 (`U+0009`) 的宽度。
+**`tab-size`** CSS 属性用于自定义制表符字符的宽度 (U+0009)。
```
Comment (yin1999) [inline files/zh-cn/web/css/tab-size/index.md:8]:
```suggestion
**`tab-size`** CSS 属性用于自定义制表符（U+0009）的宽度。
```

File: files/zh-cn/web/css/tab-size/index.md [hunk 2]
```diff
@@ -3,70 +3,72 @@ title: tab-size
...
-{{CSSInfo}}
-
-## 语法
-
-### 值
+### 取值
 
 - {{CSSxRef("&lt;integer&gt;")}}
-  - : 制表符对应的空格数。必须为非负值。
+  - : 使用空格字符（U+0020）的进阶宽度的倍数作为制表符的宽度。必须是非负数。
```
Comment (yin1999) [inline files/zh-cn/web/css/tab-size/index.md:32]: I'm not sure 'advance width' could be translated to "进阶宽度"

File: files/zh-cn/web/css/tab-size/index.md [hunk 3]
```diff
@@ -3,70 +3,72 @@ title: tab-size
...
 - {{CSSxRef("&lt;length&gt;")}}
-  - : 制表符的宽度。必须为非负值。
+  - : 制表符的宽度必须是非负数。
```
Comment (yin1999) [inline files/zh-cn/web/css/tab-size/index.md:34]: why removing the period

```suggestion
  - : 制表符的宽度。必须是非负数。
```

**Review by yin1999 — CHANGES_REQUESTED** [review]

File: files/zh-cn/web/css/tab-size/index.md (hunk 2 above)
Comment (daiwanxing) [reply files/zh-cn/web/css/tab-size/index.md:32]: 嗯，应该是指的是字符的宽度，疏漏了。https://learn.microsoft.com/en-us/windows/win32/gdi/character-widths

**Review by yin1999 — APPROVED** [review]
Thanks 👍

**Pull request merged**